askup/utils/views.py

```diff
@@ -39,9 +39,9 @@
 QSET_QUESTION_FILTERS = {
     'all': ('All', 'All the questions'),
     'mine': ('Mine', 'My question'),
-    'others': ('Others', 'The questions of the others'),
-    'unanswered': ('Unanswered', 'The questions that I didn\'t answer before'),
-    'incorrect': ('Incorrect', 'The questions that I was incorrectly answered last time'),
+    'others': ('Others', 'Other\'s questions'),
```

Comment:
I think this should be `'Others\' questions'` actually (apostrophe after the plural 's').

Reply:
Got it. Thanks.